docs/guide/redeem.md

```diff
@@ -0,0 +1,23 @@
+# Redeem
+
+In the concept of the prediction market, the outcome token we purchased will
+eventually be settled into the original token, which is ZBS in our system, but
```

Comment:
I think the App Guide should use ZTG instead of ZBS, even if that means that some of the screenshots are off.